src/components/vote/VoteModal.tsx

```diff
@@ -94,7 +94,7 @@ export default function VoteModal({ isOpen, onDismiss, proposalId, voteOption }:
                   <Trans>Vote to abstain on proposal {proposalId}</Trans>
                 )}
               </ThemedText.MediumHeader>
-              <StyledClosed stroke="black" onClick={wrappedOndismiss} />
+              <StyledClosed onClick={wrappedOndismiss} />
```

Comment:
Is this intentional?

Reply:
yeah, the button was invisible in dark mode with this prop, now it toggles with theme